iteration2_exercise24.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import re
+
+def get_two_strings():
+    """Get input from user for two strings, and normalize them"""
+
+    user_string_1 = input("Enter one string: ")
+    user_string_2 = input("Enter a second string: ")
+
+    return user_string_1.lower(), user_string_2.lower()
+
+def strings_contain_only_letters(user_string_1, user_string_2):
+    """Determine if strings contain only letters"""
+
+    string_1_all_letters = re.search(r'^[a-z]+$', user_string_1)
```

Comment:
You can actually extract this regex into it's own variable and pass it into both `search` functions:


```
strings_with_only_letters_regex = r'^[a-z]+$'

string_1_all_letters = re.search(strings_with_only_letters_regex, user_string_1)
...
```

Furthermore, this would be a case where you can `compile` the regex with `re.compile` to (slightly) speed up your program. But regardless, it's a best practice to compile a regex if you use it more than once.